### ISSUE TITLE
option to reverse encoder

### ISSUE THREAD

**Issue by sibrat** [NoobishSVK/TEF6686-remastered#3]
add option to reverse encoder direction

**Comment by NoobishSVK** [comment]
Hello, this option already exists and works the same as in the original firmware. Try to hold either the encoder button or the top button on the front side while starting up the tuner and let me know if that helps.

**Comment by sibrat** [comment]
oh, yes, it's works. Thank you. Then We need some manual...

**Comment by sibrat** [comment]
no, issue not so simple. Frequency tuning indeed change direction, but not the cursor in menus

**Comment by NoobishSVK** [comment]
Which firmware version are you trying to run this on? There were some changes after v1.21. 

When it comes to the manual, I could technically create one, however I'm just an independent developer and I do this in my free time, so it might take a while.

**Comment by sibrat** [comment]
v1.21 indeed. i have some troubles with compiling from source as some libraries not available on github anymore (https://github.com/mrfaptastic/WiFiConnectLite). and messing up with platformio is a bit too much for me

**Comment by NoobishSVK** [comment]
Sorry for the late answer, I will be publishing v1.22 soon and I'll be sure to provide a mirror for WifiConnect Lite as the original author seems to have deleted the repository from GitHub. 
Regarding your issue with the lack of a manual, that's already being worked on.
